src/adu_types/src/adu_types.c

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file adu_type.h
```

Comment:
>adu_type.h [](start = 9, length = 10)

adu_type.c

Reply:
oops... my bad, should be adu_types.c

---
In reply to: [591963544](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591963544) [](ancestors = 591963544)